pkg/config/config.go

```diff
@@ -186,6 +186,7 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("disable_py3_validation", false)
 	config.BindEnvAndSetDefault("python_version", DefaultPython)
 	config.BindEnvAndSetDefault("allow_arbitrary_tags", false)
+	config.BindEnvAndSetDefault("use_proxy_for_cloud_metadata", false)
```

Comment:
Since we talk about this in the changelog we should add it to the configuration template.

Reply:
Updated - let me know if the wording sounds good to you.